packages/crayons-core/src/components/data-table/custom-cells/user/custom-cell-user.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'fw-custom-cell-user',
+  styleUrl: 'custom-cell-user.scss',
+  shadow: true,
+})
+export class CustomCellUser {
+  @Prop() image = null;
+
+  @Prop() name = '';
+
+  @Prop() email = '';
+
+  getInitialsFromName() {
```

Comment:
I see this as a recurring pattern, Shall we move this to a util method, or better move this inside the avatar component itself?

Reply:
Moved this logic to avatar component. Introduced name property to avatar component and displaying initials from name when image and initials props are not present.